backend/src/main/java/com/redhat/cloud/notifications/routers/UserConfigResource.java

```diff
@@ -267,4 +271,10 @@ private SettingsValuesByEventType getSettingsValueForUserByEventType(List<EventT
         patchWithUserPreferencesIfExists(settingsValues, emailSubscriptions);
         return settingsValues;
     }
+
+    private static void forbidAccessInCaseOfServiceAccountAuthentication(SecurityContext sec) {
+        if (isServiceAccountAuthentication(sec)) {
+            throw new ForbiddenException("This api can't be used form a service account authentication");
```

Comment:
There's a typo in `from` but since it'll be fixed, I would use `with` instead 😄 
```suggestion
            throw new ForbiddenException("This api can't be used with a service account authentication");
```